presto-main/src/test/java/io/prestosql/sql/planner/sanity/TestDynamicFiltersChecker.java

```diff
@@ -200,6 +200,46 @@ public void testUnmatchedNestedDynamicFilter()
         validatePlan(root);
     }
 
+    @Test(expectedExceptions = VerifyException.class, expectedExceptionsMessageRegExp = "Dynamic filter expression \\(\"LINEITEM_OK\" \\+ BIGINT '1'\\) must be a SymbolReference or a CAST of SymbolReference.")
+    public void testUnsupportedDynamicFilterExpression()
```

Comment:
wasn't there a test case like that?

Reply:
i didn't come across one, if i look at the initial commit which added the check for SymbolReference in RemoveUnsupportedDynamicFilters, it relied on the test in `TestDynamicFilter#testJoinOnCast`